### ISSUE TITLE
AMO needs some fixes

### ISSUE THREAD

**Issue by stonecrusher** [overdodactyl/ShadowFox#250]
Currently (FF 65.0.1, SF 1.6.0) there are some glitches on amo:
bad fontcolor contrast, fuzzy star background, missing yellow star color.
Link example: https://addons.mozilla.org/en-US/firefox/addon/stylish/reviews/1199485/
Screenshot: ![2019-02-28](https://user-images.githubusercontent.com/1388389/53567607-077c5a00-3b60-11e9-85b7-376c1fb419a4.png)

**Comment by overdodactyl** [comment]
Thanks - let me know if you see any other issues.

**Comment by stonecrusher** [comment]
Is it intended that the stars on search results are grey? Imho not so good contrast, especially the stars-not-given.
![2019-03-04 13_56_11-suchergebnisse fur darkreader add-ons fur firefox de](https://user-images.githubusercontent.com/1388389/53735326-d618cc80-3e86-11e9-8a16-7de1166519da.png)

**Comment by jingofett** [comment]
The color of some links are very hard to read:

![11e7c862-e72c-428f-aa86-84aff1695527](https://user-images.githubusercontent.com/8313249/57342246-7ca27780-712d-11e9-8681-fd019e5c0f91.png)
